--- ncmpcpp-0.3.5/configure.in.orig	2009-07-25 14:46:22.000000000 +0200
+++ ncmpcpp-0.3.5/configure.in	2009-07-25 14:47:05.000000000 +0200
@@ -60,7 +60,7 @@
 if test "$pdcurses" = "no" ; then
 	pdcurses=XCurses
 	if test "$unicode" = "yes" ; then
-		curses_config_bin=ncursesw5-config
+		test -f /usr/bin/ncursesw6-config && curses_config_bin=ncursesw6-config || curses_config_bin=ncursesw5-config
 		curses_lib=ncursesw
 		AC_DEFINE([_UTF8], [1], [enables unicode support])
 	else
