diff -urN ncmpcpp-0.7.6.orig/src/utility/html.cpp ncmpcpp-0.7.6/src/utility/html.cpp
--- ncmpcpp-0.7.6.orig/src/utility/html.cpp	2016-10-31 23:17:38.007041389 +0100
+++ ncmpcpp-0.7.6/src/utility/html.cpp	2016-10-31 23:18:00.243521920 +0100
@@ -66,7 +66,7 @@
 	for (size_t i = s.find("<"); i != std::string::npos; i = s.find("<"))
 	{
 		size_t j = s.find(">", i)+1;
-		if (s.compare(i, std::min(3ul, j-i), "<p ") == 0 || s.compare(i, j-i, "</p>") == 0)
+		if (s.compare(i, std::min(std::size_t{3}, j-i), "<p ") == 0 || s.compare(i, j-i, "</p>") == 0)
 			s.replace(i, j-i, "\n");
 		else
 			s.replace(i, j-i, "");
